lib/renderer.js

```diff
@@ -1265,11 +1265,13 @@
             canvasDimensionsScale = 1,
             settingsPromise,
             resultPromise,
+            hasVectorMask = layer && layer.path && layer.path.enabled,
```

Comment:
Also need to clarify this in the comment/docs ... which already mention masks.  I just want this to be clear for our future-selves that will eventually be fixing this madness.